### PR TITLE
Document manual install of JetBrains IDEs

### PR DESCRIPTION
**Pull request by fioan89**
[ch5978] - Document manual install of JetBrains IDEs on coder environment

image

### CODE REVIEW

File: environments/editors.md [hunk 1]
```diff
@@ -52,6 +52,26 @@ launch the editor from the dashboard.
 > If you need a valid license to run your IDE locally, you'll also need one to
 > run it in Coder.
 
+### Install JetBrains IDEs
+
+IDEs can also be manually installed. Once you've followed JetBrains steps
```
Comment (jakelang) [inline environments/editors.md:57]: Should be `JetBrains'`

Comment (jakelang) [reply environments/editors.md:57]: Also, maybe begin with `JetBrains IDEs` rather than `IDEs`

File: environments/editors.md [hunk 2]
```diff
@@ -52,6 +52,26 @@ launch the editor from the dashboard.
 > If you need a valid license to run your IDE locally, you'll also need one to
 > run it in Coder.
 
+### Install JetBrains' IDEs
+
+JetBrains' IDEs can also be manually installed. Once you've followed JetBrains
+steps on how to install the editor, you will have to create a symlink and add
+it to the _$PATH_.
```
Comment (khorne3) [inline environments/editors.md:59]: Maybe rephrase as

> You can manually install JetBrains' IDEs. After following JetBrains' steps for installing your IDE, create a symlink and add it to _$PATH_.

?

**Review by khorne3 — APPROVED** [review]
